support/azure/virtual-machines/linux/troubleshoot-linux-rhui-certificate-issues.md

```diff
@@ -542,12 +542,38 @@ The following steps apply if the OS version is *RHEL 8.10* and the VM was create
    sudo echo $(. /etc/os-release && echo $VERSION_ID) > /etc/dnf/vars/releasever
    ```
 
-3. Verify that the corresponding repositories are available and show no errors. To do this, run the `yum repolist` command:
+3. Verify that the corresponding repositories are available and show no errors. To do this, run the `dnf repolist` command:
 
    ```bash
    sudo dnf repolist all
    ```
+  
 
+#### [RHEL 8.10 - RHEL-HA (E4S)](#tab/rhe810-rhel-base-ha)
```

Comment:
rhel 8.10 is non eus or non e4s release

Reply:
@vkchilak Update the tittle from #### [RHEL 8.10 - RHEL-HA (E4S)](#tab/rhe810-rhel-base-ha)   to  #### [RHEL 8.10 - RHEL-HA(#tab/rhe810-rhel-base-ha)

Before was added as e4s because the HA images were using e4s repos. We reported to PG and Red Hat recommended that should not by e4s anymore. that is the reason of this change.